crates/bin/pcli/tests/network_integration.rs

```diff
@@ -52,12 +53,19 @@ static UNBONDING_DURATION: Lazy<Duration> = Lazy::new(|| {
 fn load_wallet_into_tmpdir() -> TempDir {
     let tmpdir = tempdir().unwrap();
 
+    let grpc_url: Url = std::env::var("PENUMBRA_NODE_PD_URL")
+        .unwrap_or_else(|_| "http://127.0.0.1:8080".to_owned())
+        .parse()
+        .expect("failed to parse PENUMBRA_NODE_PD_URL");
+
```

Comment:
Added a sane default for which grpc node pcli should talk to, same as already exists in the other tests. This value takes precedence when no env var override exists.